webapp/channels/src/components/quick_switch_modal/quick_switch_modal.tsx

```diff
@@ -96,7 +96,7 @@ export default class QuickSwitchModal extends React.PureComponent<Props, State>
         this.focusTextbox();
     };
 
-    private onHide = (): void => {
+    private onSelect = (): void => {
```

Comment:
Instead of calling these `onHide` and `onSelect`, could you rename these methods to `hideOnCancel` and `hideOnSelect` to be more clear about how they're intended to be used. I've always found the existing names for things like `onHide` and `onExited` to be a bit confusing anyway